src/js/containers/FormPage.jsx

```diff
@@ -99,6 +99,10 @@ class FormPage extends React.Component {
       // And the data should be for just the item in the array
       data = _.get([route.pageConfig.arrayPath, params.index], data);
     }
+    // It should be "safe" to check that this is the first page because it is
+    // always eligible and enabled, no need to call getPreviousPagePath.
+    const isFirstRoutePage =
+        (route.pageConfig.urlPrefix || '/') + route.pageList[0].path === this.props.location.pathname;
```

Comment:
I think you shouldn't need to add the `urlPrefix` to the `pageList` path; the code [here](https://github.com/usds/us-forms-system/blob/master/src/js/helpers.js#L78) already does that when generating the `pageList` object to add to the `route` object. When I run this change locally using the starter-app, it's not working as expected because of that.

Reply:
Ah, thanks! So if that's the case it sounds like the unit tests are not set up correctly as you mentioned below. I didn't mean to get you into this so deeply but had hoped you might know some background about how this works that would save time. If you want I can take this back and dig into it.

Reply:
No worries at all! I needed to understand this better myself, so I learned a lot by digging in. Let me know how I can be helpful; if you want to run with making further updates to this that's great, and I can pitch in where needed!

Reply:
*(grabs baton)*